http-server/src/main/java/com/facebook/airlift/http/server/HttpServerConfig.java

```diff
@@ -693,4 +705,49 @@ public HttpServerConfig setHttp2StreamIdleTimeout(Duration http2StreamIdleTimeou
         this.http2StreamIdleTimeout = http2StreamIdleTimeout;
         return this;
     }
+
+    public boolean isAuthEnabled()
+    {
+        return authEnabled;
+    }
+
+    @Config("http-server.auth.enabled")
```

Comment:
`http-server.authorization.enabled`

Reply:
Since authorization check doesn't make much sense without authentication first. We are planning to move the authentication part from Presto to Airlift.
If so, I thought the authn and authz could share the same config `http-server.auth.enabled`. Or should we keep them separate?

Reply:
I don't feel strong, but it seems like sometimes it is useful to have authentication without authorization. @mayankgarg1990 ?

Reply:
I agree - authentication is still necessary for data security - so we should allow for both - just authentication and authn and authz combined.